### PR TITLE
fix: breaking ffi changes

### PR DESCRIPTION
**Pull request by load1n9**
Deno pointers are no longer bigints, these changes fix the breaking generated code

### CODE REVIEW

File: codegen.ts [hunk 1]
```diff
@@ -84,6 +85,7 @@ function resolveDlopenParameter(typeDefs: TypeDef, type: any): string {
   } else {
     return "pointer";
   }
+  // deno-lint-ignore no-unreachable
   throw new TypeError(`Type not supported: ${t}`);
```
Comment (lino-levan) [inline codegen.ts:89]: Maybe I'm not understanding something but typescript seems to be correct in it claiming that this is unreachable. Why is this needed?

File: cli.ts [hunk 2]
```diff
@@ -69,7 +69,7 @@ async function generate() {
 
 try {
   await Deno.remove(metafile);
-} catch (e) {
+} catch (_e) {
```
Comment (lino-levan) [inline cli.ts:72]:
```suggestion
} catch {
```

We don't need to do `_e` here.

**Review by lino-levan — CHANGES_REQUESTED** [review]
Overall looks good besides two minor nitpicks.

**Comment by lino-levan** [comment]
Wait hold on, is this a duplicate of #117? Either way, one of these two should be merged.

File: codegen.ts (hunk 1 above)
Comment (load1n9) [reply codegen.ts:89]: don't know why its there wasn't going to risk removing it if someone kept it there to add later on

**Review by lino-levan — APPROVED** [review]
LGTM. @kt3k please take a look!

**Comment by kt3k** [comment]
@littledivy PTAL

**Review by littledivy — APPROVED** [review]
LGTM

File: codegen.ts (hunk 1 above)
Comment (zifeo) [reply codegen.ts:89]: You can remove this line... It is unreachable.

**Review by zifeo — APPROVED** [review]

**Comment by zifeo** [comment]
@load1n9 Maybe have a look at open PR to consolidate efforts for the next time :)

**Comment by lino-levan** [comment]
Still waiting on this to get merged... not sure who should do that?

**Comment by kt3k** [comment]
ref: https://github.com/denoland/deno/pull/16889

**Comment by kt3k** [comment]
@littledivy Do you think the current main is ready enough to be released? If so, I'll try to cut 0.8.0

**Comment by kt3k** [comment]
Looks like `deno_bindgen --release=path` command isn't working now (probably because of `plug` upgrade)? I'm trying to fix it in #121.
